templates/tease-search.twig

```diff
@@ -25,7 +27,9 @@
 			alt="{{ fn('esc_attr', post.thumbnail_alt|default( post.post_title ) )|raw }}"
 			data-ga-category="Search Results"
 			data-ga-action="Image"
-			data-ga-label="{{ ga_page_type }}" />
+			data-ga-label="{{ ga_page_type }}"
+			onclick="window.location.href='{{ post_link }}'"
```

Comment:
If we want the image to be a clickable link, we should use an `a` tag around it.

Implementing our own link with JS misses a lot of things about the UX of links:
- focus states are different
- not accessible
- open in a new tab won't work
- probably more